Makd.mak

```diff
@@ -36,6 +36,15 @@ F ?= devel
 # use "." for the current directory)
 SRC = src
 
+# Directory where all the integration tests are
+ifndef INTEGRATIONTEST
```

Comment:
I tried defining `INTEGRATIONTEST:=test` in `Config.mak` but message keeps being printed